src/components/SelectChildForClass.tsx

```diff
@@ -0,0 +1,95 @@
+import { Flex, Box, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React, { useState, SetStateAction } from "react";
+import { CloseButton } from "@components/CloseButton";
+import colourTheme from "@styles/colours";
+
+type SelectChildForClassProps = {
+    children: string[];
+    selectedChild: number;
+    setSelectedChild: React.Dispatch<SetStateAction<number>>;
+    pageNum: number;
+    setPageNum: React.Dispatch<SetStateAction<number>>;
+};
+
+export default function SelectChildForClass(
+    props: SelectChildForClassProps,
+): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    // Test data to be replaced with children associated with parent during integration
+    // const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+    // const [selectedChild, setSelectedChild] = useState<string>("");
+
+    return (
+        <Box>
+            <Flex justifyContent="flex-end">
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/" />
+            </Flex>
+            <Center>
+                <Text align="center" mt="15px" fontWeight="700" fontSize="36px">
+                    Program Registration
+                </Text>
+            </Center>
+            <Center>
+                <Text pb="55px" align="center" mt="30px">
+                    Who would you like to register for{" "}
+                    <b>
+                        Building Bridges with Music - <br />
+                        Singing Monkeys (Ages 9 and under)?{" "}
+                    </b>
+                    (select one)
+                </Text>
+            </Center>
+
+            <Center>
+                <VStack spacing={5}>
+                    {props.children.map((childName, index) => (
+                        <Button
+                            _focus={{ boxShadow: null }}
+                            _hover={{ border: "2px solid #0C53A0" }}
```

Comment:
I think you get can away with:
```suggestion
                            _hover={{ border: `2px solid ${colourTheme.colors.Blue}` }}
```